app/org/sagebionetworks/bridge/StudyConsentBootstrapper.java

```diff
@@ -37,6 +37,7 @@ public StudyConsentBootstrapper(StudyService studyService, StudyConsentDao study
             study.setMinAgeOfConsent(18);
             study.setResearcherRole("api_researcher");
             study.setConsentNotificationEmail("bridge-testing+consent@sagebridge.org");
+            study.setSupportEmail("bridge-testing+support@sagebase.org");
```

Comment:
It is referenced in one of Eric's tests, but hadn't been set in my local environment, so I added this to get his test to pass. I wonder if bridge-testing@sagebase.org is sufficient to make these bridge-testing emails work?